shell/plugins/index.js

```diff
@@ -4,8 +4,14 @@
  * These are included in a function that can be explictly called, so that we can be sure
  * of the execution order, rather than importing them at the top of a file.
  */
-export function loadDirectives() {
-  import('./clean-html-directive');
-  import('./clean-tooltip-directive');
-  import('./directives');
+import cleanHtmlDirective from '@shell/plugins/clean-html-directive';
+import vCleanTooltip from '@shell/plugins/clean-tooltip-directive';
+import focus from '@shell/plugins/directives';
+import trimWhitespaceDirective from '@shell/plugins/trim-whitespace';
+
+export function loadDirectives(Vue) {
+  Vue.use(cleanHtmlDirective);
```

Comment:
There's nothing inherently wrong with writing our directives as vue plugins and loading them via `Vue.use()`, but we can also consider if `Vue.directive()` will work here.